docusaurus/docs/quickstarts/network/hosted/index.md

```diff
@@ -1,168 +1,258 @@
+
+$ createRouterSystemdFile "${ZITI_EDGE_ROUTER_RAWNAME}"
+Router systemd file written to: /home/ubuntu/.ziti/quickstart/ip-172-31-23-18/ip-172-31-23-18-edge-router.service
+```
+
+After the files are generated, you can then install them for use by systemd by running:
+
+```bash
+# run me
+sudo cp "${ZITI_HOME}/${ZITI_EDGE_CONTROLLER_RAWNAME}.service" /etc/systemd/system/ziti-controller.service
+sudo cp "${ZITI_HOME}/${ZITI_EDGE_ROUTER_RAWNAME}.service" /etc/systemd/system/ziti-router.service
+sudo systemctl daemon-reload
+sudo systemctl enable --now ziti-controller
+sudo systemctl enable --now ziti-router
+```
+
+```bash
+# example output
+$ sudo cp "${ZITI_HOME}/${ZITI_EDGE_CONTROLLER_RAWNAME}.service" /etc/systemd/system/ziti-controller.service
+
+$ sudo cp "${ZITI_HOME}/${ZITI_EDGE_ROUTER_RAWNAME}.service" /etc/systemd/system/ziti-router.service
+
+$ sudo systemctl daemon-reload
+
+$ sudo systemctl enable --now ziti-controller
+Created symlink from /etc/systemd/system/multi-user.target.wants/ziti-controller.service to /etc/systemd/system/ziti-controller.service.
+
+$ sudo systemctl enable --now ziti-router
+Created symlink from /etc/systemd/system/multi-user.target.wants/ziti-router.service to /etc/systemd/system/ziti-router.service.
+```
+
+Now, both the controller and the edge router will restart automatically!  After a few seconds you can then run these
+commands and verify systemd has started the processes and see the status:
+
+```bash
+# run me
+sudo systemctl -q status ziti-controller --lines=0 --no-pager
+sudo systemctl -q status ziti-router --lines=0 --no-pager
+```
+
+```bash
+# example output
+$ sudo systemctl -q status ziti-controller --lines=0 --no-pager
+● ziti-controller.service - Ziti-Controller
+     Loaded: loaded (/etc/systemd/system/ziti-controller.service; disabled; vendor preset: enabled)
+     Active: active (running) since Thu 2021-11-11 19:05:46 UTC; 8s ago
+   Main PID: 2375 (ziti-controller)
+      Tasks: 7 (limit: 1154)
+     Memory: 43.7M
+     CGroup: /system.slice/ziti-controller.service
+             └─2375 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-controller run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/co…
+
+$ sudo systemctl -q status ziti-router --lines=0 --no-pager
+● ziti-router.service - Ziti-Router for ip-10-0-0-1-edge-router
+     Loaded: loaded (/etc/systemd/system/ziti-router.service; disabled; vendor preset: enabled)
+     Active: active (running) since Thu 2021-11-11 19:05:47 UTC; 8s ago
+   Main PID: 2385 (ziti-router)
+      Tasks: 6 (limit: 1154)
+     Memory: 231.4M
+     CGroup: /system.slice/ziti-router.service
+             └─2385 /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ziti-bin/ziti-v0.22.11/ziti-router run /home/ubuntu/.ziti/quickstart/ip-10-0-0-1/ip-10…
+```
+
+Inspect the servers' listening ports.
+
+```bash
+# run me
+sudo ss -lntp | grep ziti
+```
+
+```bash
+# example
+$ sudo ss -lntp | grep ziti
```

Comment:
I don't have `ss` available on Mac, I'm not sure if it's standard in Linux. In the past, I've used `lsof` which does ship with Mac but I'm also not sure if that is standard on Linux.

Reply:
Do you think we should say the hosted anywhere quickstart works on macOS? Seems like the local quickstart is a better fit for workstations. It makes sense for this one to be aimed squarely at Linux servers because of the need to open firewall ports and such, the intention to make the controller and router publicly reachable.